aeppl/tensor.py

```diff
@@ -10,50 +10,50 @@
 from aesara.tensor.random.op import RandomVariable
 from aesara.tensor.random.rewriting import local_dimshuffle_rv_lift, local_rv_size_lift
 
-from aeppl.abstract import MeasurableVariable, assign_custom_measurable_outputs
+from aeppl.abstract import (
+    MeasurableVariable,
+    ValuedVariable,
+    assign_custom_measurable_outputs,
+)
 from aeppl.logprob import _logprob, logprob
-from aeppl.rewriting import PreserveRVMappings, measurable_ir_rewrites_db
+from aeppl.rewriting import measurable_ir_rewrites_db
 
 
 @node_rewriter([BroadcastTo])
-def naive_bcast_rv_lift(fgraph, node):
-    """Lift a ``BroadcastTo`` through a ``RandomVariable`` ``Op``.
-
-    XXX: This implementation simply broadcasts the ``RandomVariable``'s
-    parameters, which won't always work (e.g. multivariate distributions).
+def valued_var_bcast_lift(fgraph, node):
```

Comment:
Naive question: why does this only return a `RandomVariable`, and not a new `ValuedVariable`? Also, why are we not setting the shape of the associated value variables as well?

Reply:
I don't think we even need that rewrite anymore, so I'll remove it.